Sources/Target/FreeBSD/Process.cpp

```diff
@@ -201,8 +201,7 @@ ErrorCode Process::wait() {
 
   switch (_currentThread->_stopInfo.event) {
   case StopInfo::kEventNone:
-    switch (_currentThread->_stopInfo.reason) {
-    case StopInfo::kReasonNone:
+    if (_currentThread->_stopInfo.reason == StopInfo::kReasonNone) {
```

Comment:
Yeah I guess if we can't test it it makes sense to keep it like that. Works for me.
I suppose the errors you were dealing with were for incomplete switch?